### PR TITLE
feat: add um-idcard

### PR DESCRIPTION
**Pull request by mockdot**
## Description

- Add um-idcard
- Tiny bit of more code cleanup

## Checklist

- [ ] I have personally loaded this code into an updated Qbox project and checked all of its functionality.
- [x] My code fits the style guidelines.
- [x] My PR fits the contribution guidelines.

### CODE REVIEW

File: server/main.lua [hunk 1]
```diff
@@ -143,8 +109,6 @@ RegisterNetEvent('qb-cityhall:server:ApplyJob', function(job)
     TriggerClientEvent('QBCore:Notify', src, Lang:t('info.new_job', {job = JobInfo.label}), 'success')
 end)
 
-RegisterNetEvent('qb-cityhall:server:getIDs', giveStarterItems)
-
```
Comment (Manason) [inline server/main.lua:147]: We're not giving the id card and driver license as starting items anymore?

**Review by Manason — CHANGES_REQUESTED** [review]
Should we add um-idcard in fxmanifest dependencies?

File: server/main.lua (hunk 1 above)
Comment (mockdot) [reply server/main.lua:147]: It doesn't give them from here, but from multichar

**Comment by StrixLuca** [comment]
Yes is a good script

**Comment by StrixLuca** [comment]
finally someone who listens

**Pull request closed without merging**